tests/e2e-instrumentation/instrumentation-java/01-assert.yaml

```diff
@@ -17,14 +17,19 @@ spec:
       valueFrom:
         fieldRef:
           fieldPath: status.podIP
+    - name: JAVA_TOOL_OPTIONS
+      valueFrom:
+        configMapKeyRef:
+          name: config-java
+          key: system-properties
     - name: OTEL_JAVAAGENT_DEBUG
       value: "true"
     - name: OTEL_INSTRUMENTATION_JDBC_ENABLED
       value: "false"
     - name: SPLUNK_PROFILER_ENABLED
       value: "false"
     - name: JAVA_TOOL_OPTIONS
-      value: ' -javaagent:/otel-auto-instrumentation-java/javaagent.jar'
+      value: '$(JAVA_TOOL_OPTIONS) -javaagent:/otel-auto-instrumentation-java/javaagent.jar'
```

Comment:
are we able to verify that this works? I didn't think that additive variables with the same name work?

Reply:
(verify in the test)

Reply:
it works, we are even using it for getting the pod name and namespace to resource attributes env var

Reply:
https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/

Reply:
I think Jacob was asking if defining the same variable twice works as one would intuitively expect here.

Reply:
I have added the link into the source code. 

As I mention we already use it for other env vars